presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -871,6 +872,22 @@ public void testCreateViewColumns()
     }
 
     @Test
+    public void testCreateRecursiveView()
+            throws Exception
+    {
+        assertFails(VIEW_IS_RECURSIVE, "CREATE OR REPLACE VIEW v5 AS SELECT * FROM v5");
```

Comment:
This test is confusing since `v5` is already recursive. Make this test try to replace `v1` (which is a normal, valid view).